get_geojson_objects.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import os
+
+"""
+This is a simple example of a script one might use to get geojson objects from a specific farm.
+In the script, we find a farm ID based on the farm name, pull an informational summary about the farm, and use that summary data to iterate through the geojson request.
+
+You will need to provide a bearer token in order to run this script. You can find your bearer token by following the directions in the API docs found at https://docs.raptormaps.com/reference/get-api-access-token.
+Additionally, you will need to know your organizational ID and the name of the farm you are interested in.
+"""
+
+
+bearer_token = os.environ['BEARER_TOKEN']
+auth_header = {"authorization": "Bearer {}".format(bearer_token)}
+base_url = 'https://api.raptormaps.com'
+org_id = <YOUR_ORG_ID>
+
+# get farm_id from /sorted_solar_farms response body by querying by name
+def get_farm_id(farm_name):
+	res = requests.get(base_url+'/sorted_solar_farms?org_id={}&name={}'.format(org_id, farm_name), headers=auth_header)
```

Comment:
I think it's a little cleaner to use fstrings: https://docs.python.org/3/tutorial/inputoutput.html